Automattic/wp-calypso
# WooCommerce: Add setup pages step to the setup process.

Fixes #14867.

This adds an additional step to the store setup process, where we create some pages that are needed for WooCommerce to function correctly.

To Test:
* **NOTE**: Until D6219 is merged, you will need to sandbox the REST API for this to work properly, otherwise the step will never advance.
* Go to `https://developer.wordpress.com/docs/api/console/` and set your flags to 0 to start the setup process over.
* Go to `http://calypso.localhost:3000/store/:site`
* After the plugin install step, a page setup step will appear. You should automatically advance once this step finishes.
* Go to `wp-admin` (or Calypso pages) and verify that pages were created, with shortcode contents like `[woocommerce_checkout]`.
* Run `npm run test-client client/extensions/woocommerce/state` and make sure all tests pass.
